README.md

```diff
@@ -83,6 +83,13 @@ pip install -r requirements.txt
 curl "https://test-videos.co.uk/vids/jellyfish/mp4/h264/720/Jellyfish_720_10s_5MB.mp4" --output jellyfish720p.mp4
 ```
 
+### [Optional]Network Time Protocol(NTP)
+
+Since our current command system design is based on the Raspberry Pi timestamp, synchronizing the system time of these Raspberry Pi's in a local network of multiple Raspberry Pi's is a prerequisite.
```

Comment:
isn't this a contradiction? This section is called `[Optional]` but the text here says it is a `prerequisite`

Reply:
Changed.